systemd/server.service

```diff
@@ -4,7 +4,7 @@ Description=umwelt.info server service
 [Service]
 User=umwelt-info
 Group=umwelt-info
-Environment=RUST_LOG=info DATA_PATH=/var/lib/umwelt-info BIND_ADDR=0.0.0.0:8080
+Environment=RUST_LOG=info DATA_PATH=/var/lib/umwelt-info BIND_ADDR=0.0.0.0:8080 REQUEST_LIMIT=512
```

Comment:
This number is completely made up and should be specified based on load tests using our target virtual machine.